CHANGELOG.md

```diff
@@ -7,6 +7,13 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [Unreleased]
 
+## [v0.3.3] - 2020-11-17
```

Comment:
Missing the v0.3.3 change link at the bottom of the CHANGELOG.md for the release.

Reply:
ah, didn't see that, but I'll fix it

Reply:
done